scripts/docs/metric-doc-to-agent

```diff
@@ -22,8 +22,8 @@ for f in ${INTEGRATIONS_REPO%/}/$INTEGRATION/docs/*.md; do
     continue
   fi
   metric_name=$(basename $f .md)
-  desc=$(head -n4 $f | yq r - 'brief')
-  type=$(head -n4 $f | yq r - 'metric_type')
+  desc=$(head -n4 $f | yq -r ."brief")
+  type=$(head -n4 $f | yq -r ."metric_type")
```

Comment:
The yq command syntax is dependent on whether you have yq or python-yq installed on your system. Maybe we should try to handle both, or only allow one or the other?

Reply:
Yeah, I would rather not change this, but would rather update the dev image to install the right version of yq that works with this.

Reply:
> Yeah, I would rather not change this, but would rather update the dev image to install the right version of yq that works with this.

This script seems to be intended to run directly on the local system (outside of the dev image) since it also requires the integrations repo.

Reply:
Ok, reverted scripts/docs/metric-doc-to-agent